pkg/util/apikey/balancer.go

```diff
@@ -0,0 +1,197 @@
+package apikey
+
+import (
+	"math/rand"
```

Comment:
Using math/rand/v2 is suggested. GOSEC identifies the use of math/rand as a potential risk because its output is predictable if the seed is known. (G404)

Reply:
fixed